ios/fastlane/Fastfile

```diff
@@ -199,8 +171,9 @@ platform :ios do
 
   desc "Add devices"
   lane :devices do |options|
-    new_devices = get_unprovisioned_devices_from_hockey(app_bundle_id:'org.prideinlondon.festival', api_token:options[:api_token])
-    register_devices(devices: new_devices)
+    register_devices(devices: {
+      "Example device name" => "Example UUID",
```

Comment:
See for example https://docs.fastlane.tools/actions/register_devices/#4-examples

Reply:
worth adding that as a code comment?